content/news/2025-01-03-abstract-review/index.en.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Volunteer as an R-Ladies Abstract Reviewer!"
+author: "Abstract Review Team"
+type: blog
+date: "2025-01-03"
+output: html_document
+image: img/global_team.png
+tags: 
+- community
+categories: 
+- R-Ladies
+summary: Volunteer as an R-Ladies Abstract Reviewer to support diversity in tech!
+slug: /news/abstract-review-2025/
+---
+
+Are you passionate about supporting diversity in tech and data science? Want to help aspiring speakers craft compelling abstracts for conferences? Volunteer to be a reviewer for the R-Ladies Abstract Review System and make a difference in creating a more inclusive and representative speaker lineup at key industry events!
+
+## What is the R-Ladies Abstract Review System?
+
+Our abstract review system connects aspiring speakers with experienced reviewers who can provide feedback and support. By volunteering as a reviewer, you'll play a critical role in helping individuals from underrepresented groups take their first steps into public speaking or advance their speaking careers.
+
+## Why Join as a Reviewer?
+
+* Make an Impact: Help new voices shine by sharing your expertise and guidance.
+* Flexible Commitment: Sign up now, and we'll reach out when abstracts come in.
+* Support a Global Movement: Contribute to R-Ladies' mission of promoting diverse voices in the R community.
+* Minimal Time Commitment: Totally flexible, depending on the level of feedback you wish to provide and number of abstracts you wish to review.
+
```

Comment:
* Expand your network: Meet new people from around the world and learn about their work.

Reply:
Added!